packages/odyssey-react-mui/src/themes/odyssey/theme.ts

```diff
@@ -24,7 +27,26 @@ import "./components.types";
 export const theme = createTheme({
   palette,
   shape,
+  mixins,
   spacing,
   typography,
   components,
+  odyssey: {
+    borderWidth: "1px",
+  },
 });
+
+// This TS mod allows for `odyssey` above, but not within components.ts
+declare module "@mui/material/styles" {
```

Comment:
This works _here_. Adding this mod removes any errors from `theme` above when adding `odyssey`.

Reply:
However, we still get the error seen in `components.ts`.

Changing this to `BaseTheme` does not solve the error in `components.ts`.

Changing the next mod to `BaseThemeOptions` will create errors in this file.